index.js

```diff
@@ -1535,7 +1555,13 @@ module.exports = class Autobase extends ReadyResource {
         await core.get(length - 1, { timeout })
       }
 
-      const system = new SystemView(core.session(), length)
+      const system = new SystemView(
```

Comment:
Note: this was added with the last commit (had missed this usage of `new SystemView` on my initial pass, so it wasn't yet passing through the `maxCacheSize`)